evaluators/embedding/L3NormEvaluator/README.md

```diff
@@ -0,0 +1,3 @@
+# L1NormEvaluator
```

Comment:
I'm confused. Is it L1 or L3? the folder is called 'L3...', but there are references to L1 throughout

Reply:
Sorry, I forgot to change them. Now should be fixed.